Assets/Scripts/Map/RobotSpawners/ExplorationRobotSpawner.cs

```diff
@@ -0,0 +1,9 @@
+using Maes.Algorithms;
+
+namespace MAES.Map.RobotSpawners
+{
+    public class ExplorationRobotSpawner : RobotSpawner<IExplorationAlgorithm>
+    {
+

```

Comment:
empty class?
maybe add todo?

Reply:
Empty is fine, this means it uses the default defined in the base. It's optional to overwrite the default behavior, which the patrolling robot spawner does